charts/prometheus-node-exporter/Chart.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 4.36.0
+version: 4.36.1
```

Please, raise the minor version instead.
```suggestion
version: 4.37.0
```